sdk/spring/spring-cloud-azure-integration-tests/src/test/resources/application-eventhubs-binder.yml

```diff
@@ -58,6 +58,7 @@ spring:
       bindings:
         consume-in-0:
           destination: ${AZURE_EVENTHUB_NAME_FOR_BINDER_MESSAGE}
+          error-handler-definition: consumeError
```

Comment:
Could you link the doc for this error handler def?

Reply:
https://docs.spring.io/spring-cloud-stream/docs/4.0.0-M5/reference/html/spring-cloud-stream.html#_handle_error_messages